### PR TITLE
Update cdnjs/ajax/libs/onsen/package.json

### PR DESCRIPTION
**Pull request by asial-matagawa**
Auto-update on `onsen` seems to have not been working since `2.0.0-beta.2` (latest is `2.0.4`)
because `onsen` no longer provides generated files on the Git repository since `2.0.0-beta.3`.
So I have updated `cdnjs/ajax/libs/onsen/package.json`.

If there is no problem, could you merge this PR?

----
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/OnsenUI/OnsenUI
 * Official website (optional, not the repository): https://onsen.io/
 * NPM package url (optional): https://www.npmjs.com/package/onsenui
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3936
   - Count of watchers: 254
   - Count of forks: 537
 * NPM download stats (optional):
   - Downloads in the last day: 286
   - Downloads in the last week: 1142
   - Downloads in the last month: 4149

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/onsen/package.json [hunk 1]
```diff
@@ -23,17 +23,14 @@
...
-      }
-    ]
-  }
+  "npmName": "onsenui",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/*",
+        "css/*"
```
Comment (PeterDaveHello) [inline ajax/libs/onsen/package.json]: I think we can use `"+(js|css)/**/*onsen*"`

**Comment by asial-matagawa** [comment]
@PeterDaveHello 
Thank you for the prompt reply. Sorry, I will prepare another branch.

File: ajax/libs/onsen/package.json (hunk 1 above)
Comment (asial-matagawa) [inline ajax/libs/onsen/package.json]: Thank you, but `onsenui.css` has `@import` such as `@import url("font_awesome/css/font-awesome.min.css");`, so I think `"+(js|css)/**/*` is the correct value in this case.

**Comment by PeterDaveHello** [comment]
@asial-matagawa Thanks!

**Pull request closed without merging**